src/runtime/entity.ts

```diff
@@ -231,10 +230,10 @@ class EntityInternals {
       rawData: this.dataClone()
     };
     if (this.hasCreationTimestamp()) {
-      serializedEntity.creationTimestamp = this.creationTimestamp;
+      serializedEntity.creationTimestamp = this.creationTimestamp.getTime().toString();
     }
     if (this.hasExpirationTimestamp()) {
-      serializedEntity.expirationTimestamp = this.expirationTimestamp;
+      serializedEntity.expirationTimestamp = this.expirationTimestamp.getTime().toString();
```

Comment:
why not store it as a number instead of string?

Reply:
changed to number, thanks